nimare/tests/new_json_format/dset_study1.json

```diff
@@ -0,0 +1,75 @@
+{
+  "Studies": [
+    {
+      "Groups": [
+        {
+          "@id": "gg_id1",
+          "Name": "group1",
+          "N": 12
+        }
+      ],
+      "Conditions": [
+        {
+          "@id": "cc_id1",
+          "Name": "household items"
+        },
+        {
+          "@id": "cc_id2",
+          "Name": "animals"
+        }
+      ],
+      "@id": 9593960,
+      "Analyses": [
+        {
+          "Images": {
+            "Elements": [
+              {
+                "Path": "orig/9593960_1_beta.nii.gz",
+                "Value": "Beta"
+              },
+              {
+                "Path": "orig/9593960_1_se.nii.gz",
+                "Value": "StandardError"
+              },
+              {
+                "Path": "orig/9593960_1_z.nii.gz",
+                "Value": "ZStatistic"
+              }
+            ],
+            "Space": "MNI_2mm"
+          },
+          "Points": {
+            "Elements": [
+              {
+                "@id": "pp_id1",
+                "Coordinate":[ 63.0, 65.0, 1.0],
+                "Clusterid": 1
+              }
+            ],
+            "Space": "MNI"
+          },
+          "Surfaces": {
+            "Elements": [
+              {
+                "Path": "orig/9593960_1_beta.gii.gz",
+                "Value": "Beta"
+              },
+              {
+                "Path": "orig/9593960_1_se.gii.gz",
+                "Value": "StandardError"
+              },
+              {
+                "Path": "orig/9593960_1_z.gii.gz",
+                "Value": "ZStatistic"
+              }
+            ],
+            "Space": "MNI_2mm"
```

Comment:
Probably doesn't matter for example's sake, but is MNI 2 mm a sensible surface space? Doesn't this only work for volumes?